### DISCUSSION TITLE
Add padding to unordered list

### DISCUSSION THREAD

**Discussion by regseb** [npm/feedback#326] [Website] [answered]
Hello,

In the markdown display of _README_, ordered lists (`ol`) have a left padding, but unordered lists (`ul`) do not. Example with the [`cronnor` package](https://www.npmjs.com/package/cronnor#expression-cron).

The ordered and unordered lists should have the same left padding.

**Comment by MylesBorins** [comment, marked as answer]
Hey @regseb thanks for reporting!

i've opened a ticket with the engineering team to take a look at this. In general smaller UI fixes like this can take a bit of time to get prioritized as our teams have a pretty full plate. Will update when we can fix this

**Reply by MylesBorins** [reply]
A quick update on this. We are not going to be fixing this padding, it is by design.
